lib/enyo/lib/default-templates/default-app/.gitignore

```diff
@@ -1,2 +1,4 @@
 .enyocache
-dist/
\ No newline at end of file
+dist/
```

Comment:
Hmm, good point.  We don't actually have any gulp/grunt commands in there and no dependencies.